### DISCUSSION TITLE
Expect Artillery engines playwright

### DISCUSSION THREAD

**Discussion by jbarata0** [artilleryio/artillery#1637] [Artillery]
Hi!! I want to use the artillery playwright engine to load test a site . can artillery validate that xpath has appeared ? can u give me an example code for that case ?Thanks.

**Comment by hassy** [comment]
hi @jbarata0 👋 yes you should be able to do it with Playwright's XPath selectors and `toBeVisible()` expectation.

https://playwright.dev/docs/selectors#xpath-selectors

https://playwright.dev/docs/test-assertions#locator-assertions-to-be-visible

**Comment by jbarata0** [comment]
but when i use expect like this : 
```
module.exports = { helloFlow };
import { expect } from "@playwright/test"

async function helloFlow(page) {

  await page.goto('https://si.pbsi.id/index.php');
  const locator = page.locator('xpath=//h2[contains(text(),"Sistem Informasi PBSI")]');
  await expect(locator).toBeVisible();

}
```
There's error in terminal 
```
import { expect } from "@playwright/test"
^^^^^^

SyntaxError: Cannot use import statement outside a module
```
can you help me ? @hassy

**Reply by trickpaislearning** [reply]
Hi @jbarata0 
instead of using import statement, use require like below
```js
const { expect } = require("@playwright/test");
```
